html_css_dom/the_internet/README.md

```diff
@@ -0,0 +1,183 @@
+
+Status is the `HTTP status code` like we talked about above. `Type` tells you what the response contains. 
+
+> What do you think all the different `Type` values mean?
+
+If you click on a row, you'll see more detailed information about that request and response. The main thing to look at here is the `Headers` tab. Each request contains descriptive information telling the server what to expect. You can see the request `Method` like we talked about above, and lots of other information that probably doesn't make a ton of sense right now. We'll get into what some of the headers mean in later lessons.
+
+### You do: Different website requests
+
+Pick 3 different websites that you know. Type them into the URL bar and see how many network requests each one takes to load.
+
+Scroll through the requests and see if anything comes out at you that surprises you. Write it down! We'll talk about it together.
+
+## Okay let's actually make requests
+
+Hopefully you have [Insomnia](https://insomnia.rest/download/) installed already. If not, take a minute to download and install Insomnia Core.
+
+We're going to be working with an API today called `PokeAPI`. We haven't exactly talked about what an API is yet, but just think of it as a server that responds to our requests for data. `PokeAPI` specifically contains all kinds of information about pokemon, and we can access it in various ways just by making requests.
+
+Open `Insomnia` and you'll see a screen like this:
+
+![Insomnia Homepage](./assets/insomnia-1.png)
+
+* Click `New Request`
+* Give it any name (`pokemon` if you can't think of one)
+* In the top bar next to the GET dropdown, put this url: `https://pokeapi.co/api/v2/pokemon/ditto`
+* Click **Send**
+
+> What do you see happen?
+
+What we get back is a response, which comes in the form of **JSON**. JSON stands for JavaScript Object Notation, and it's a structured format for sending and receiving data. Luckily, because we work exclusively in JSON, we can treat it just like one big giant object.
+
+Hooray! You've made your first request!
+
+### Curl
+
+Insomnia is a really nice user interface that makes requests for us, but we can also use the command line. Specifically, there's a program called `curl` that comes in built-in to MacOS.
+
+Type this in your terminal:
+
+```bash
+$ curl https://pokeapi.co/api/v2/pokemon/ditto
+```
+
+You'll get back a whole bunch of text! It's actually JSON, just unformatted in your terminal. 
+
+### Experiment!
+
+Try making requests to various URLs and see what happens! Make a request in Insomnia and then using `curl`. See how they differ. 
+
+* `https://google.com`
+* `https://swapi.dev/api`
+* `https://icanhazdadjoke.com`
+
+Explore the PokeAPI and SWapi documentation and you can see what kinds of requests you can make. Every request should be a GET, you'll just change the URL that you make the request to.
```

Comment:
Do they need more guidance than this? For example:
- a goal (find X different pokemon with the API, find X different types of resources you can request from the API)
- example URLs to try
- different ways to try changing the URL
- links to the API docs

Reply:
This is something I would probably explain during the lesson, but i could also clarify.